web/modules/custom/dbcdk_community/src/Profile/HasCommunityRolesTrait.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * @file
+ * Definition of HasCommunityRolesTrait.
+ */
+
+namespace Drupal\dbcdk_community\Profile;
+
+
+/**
+ * This trait is useful when extending model classes related to community roles.
+ */
+trait HasCommunityRolesTrait {
+
+  /**
+   * The community roles related to this object.
+   *
+   * @var \DBCDK\CommunityServices\Model\CommunityRole[]
+   */
+  protected $communityRoles = [];
+
+  /**
+   * Get the community roles related to this object.
+   *
+   * @return \DBCDK\CommunityServices\Model\CommunityRole[]
+   *   Related community files.
+   */
+  public function getCommunityRoles() {
+    return $this->communityRoles;
+  }
+
+  /**
+   * Set the community roles related to this object.
+   *
+   * @param |DBCDK\CommunityServices\Model\CommunityRole[] $community_roles
```

Comment:
Scrutinizer caught this: the namespace should start with a backslash, not a pipe